assets/openshift-apiserver/openshift-apiserver-deployment.yaml

```diff
@@ -72,6 +72,31 @@ spec:
 {{ end }}
       terminationGracePeriodSeconds: 90
       containers:
+{{- if .KonnectivityEnabled }}
+      - name: konnectivity-proxy
```

Comment:
This container doesn't have any readiness or liveness probes

Reply:
I can add some to test `localhost:10080`.